LabExT/Experiments/StandardExperiment.py

```diff
@@ -242,15 +242,20 @@ def run(self):
             data["sweep_information"]["sweep_association"] = OrderedDict()
             if current_todo.part_of_sweep:
                 sweep_params = current_todo.sweep_parameters
-                # we select all columns except 'finished and 'file_path'
-                for _, row in sweep_params.loc[
-                    :, (sweep_params.columns != "finished") & (sweep_params.columns != "file_path")
-                ].iterrows():
+                for _, row in sweep_params.iterrows():
                     # iterate through the rows (i.e. Measurements) belonging to this sweep and store
                     # to metadata
-                    data["sweep_information"]["sweep_association"][row["name"]] = {
-                        name: value for name, value in zip(row.index, row)
+                    temp_dict = dict()
+                    temp_dict["metadata"] = {
+                        name: value
+                        for name, value in zip(row["metadata"].index, row["metadata"])
+                        if name not in ["finished", "name"]
```

Comment:
why do you need the if statement here? What happens if we have an actual metadata called "name"? Let's discuss in tomorrow's meeting.